src/components/prompts/ShareExternal.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="stepper" id="stepper">
+    <horizontal-stepper
+      class="stepper"
+      :steps="demoSteps"
+      :locale="$t('settings.langName')"
+      @completed-step="completeStep"
+      :top-buttons="false"
+      :isRTL="$t('settings.langName')==='he'"
+      @active-step="isStepActive"
+      @stepper-finished="onStepperFinished"
+    ></horizontal-stepper>
+  </div>
+</template>
+
+<script>
+import HorizontalStepper from "vue-stepper-sh";
+import StepOne from "../files/StepOne.vue";
+import StepTwo from "../files/StepTwo.vue";
+import StepThree from "../files/StepThree.vue";
+
+import { mapState } from "vuex";
+
+
+export default {
+  name: "app",
+  components: {
+    HorizontalStepper,
+  },
+  data() {
+    return {
+      demoSteps: [
+        // StepOne - choose users to share to
+        {
+          icon: "mail",
+          name: "first",
+          title: this.$t("prompts.searchUser"),
+          component: StepOne,
+          completed: false
+        },
+        // StepTwo - choose approvers for the share
+        {
+          icon: "payment",
+          name: "second",
+          title: this.$t("prompts.searchApprover"),
+          component: StepTwo,
+          completed: false
+        },
+        // StepThree - add information and classification
+        {
+          icon: "announcement",
+          name: "third",
+          title: this.$t("exShare.headerTS"),
+          component: StepThree,
+          completed: false
+        }
+      ],
+      activeStep: 0
+    };
+  },
+  computed: {
+    ...mapState(["selected", "req"]),
+    selectedItem() {
+      return this.req.items[this.selected[0]];
+    }
+  },
+  methods: {
+    completeStep(payload) {
+      this.demoSteps.forEach(step => {
+        if (step.name === payload.name) {
+          step.completed = true;
+        }
+      });
+    },
+    isStepActive(payload) {
```

Comment:
bad name for the function

Reply:
Not sure about that. what would you suggest?

Reply:
A function name which starts with `is`, usually returns a Boolean value, which this function does not.
I would suggest `activateStep` by looking at line 10. But I'm still not completely what's the function's purpose.

Reply:
You are right. fixed it in #140